makefiles/logcleaner.mk

```diff
@@ -0,0 +1,39 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+ifeq (,$(findstring darwin,$(MEMO_TARGET)))
```

Comment:
Why is this here? The package doesn't need this restriction from what I can tell.

Reply:
copy-pasted from shshd

Reply:
You don't need it.

Reply:
macOS doesn't have `/var/mobile/`, so this is definitely (i|tv|watch)OS specific.